.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python 
+
+python: 
+  - "3.6"
+
+services: 
+  - postgresql
+
+addons:
+  postgresql: "9.6"
+
+env: 
+  -DJANGO=1.10 DB=postgresql
+
+install: 
+  - pip install -r requirements.txt
```

Comment:
I've noticed you've included a pipfile hence the assumption that we are using pipenv instead of requirements.txt. If so, I think it would be best if we install pipenv instead of requirements.txt

Reply:
I have understood. The group has decided to use virtualenv. I will push a commit removing the pipfile.

Reply:
Also please note that at the time of pushing my changes, Travis is down so the test is currently being queued indefinitely.

Reply:
I discussed the coveralls issue with my team member who integrated it. Since the command to run coveralls comes after a script has been run and we currently have no scripts or tests we agreed to leave out the coveralls from the Travis file until we have actual tests to use.

Reply:
@Kyppy I'll go ahead and merge this but will hold you to this. Once we have tests in, I trust you will add the coveralls script in the .yml file.